### PR TITLE
Add translation keys for modal creation

### PR DESCRIPTION
**Pull request by xinyihl**
Is there any specification for translating id？

Related to modrinth/code#1482

### CODE REVIEW

File: locales/en-US/index.json [hunk 1]
```diff
@@ -254,6 +254,45 @@
...
+    "message": "URL"
+  },
+  "component.modal.modal-creation.visibility.label": {
+    "message": "Visibility"
+  },
+  "component.modal.modal-creation.visibility.description": {
+    "message": "The visibility of your project after it has been approved."
+  },
+  "component.modal.modal-creation.visibility.placeholder": {
+    "message": "Choose visibility.."
```
Comment (notlin4) [inline locales/en-US/index.json:279]: I think it is missing a dot.

Comment (xinyihl) [reply locales/en-US/index.json:279]: I don't think so.
there were only two dots.

Comment (notlin4) [reply locales/en-US/index.json:279]: Other placeholders all use three dots.

Comment (xinyihl) [reply locales/en-US/index.json:279]: aright，i will fix it

**Pull request closed without merging**